modules/staqAnalyticsAdapter.md

```diff
@@ -0,0 +1,23 @@
+# Overview
+Module Name: STAQ Analytics Adapter
+
+Module Type: Analytics Adapter
+
+Maintainer: dev@staq.com
+
+# Description
+
+Analytics adapter for STAQ. Contact support@staq.com for information.
+
+# Test Parameters
+
+```
+{
+  provider: 'staq',
+  options: {
+    host: , // HOST URL of site. Optional. Only required for whitelisting.
+    url: 'localhost:3000', // REQUIRED host URL for delivery of information to STAQ
```

Comment:
@sumit116 thanks for your review.

We don't currently have an outside facing url that we leave standing.  Would it be possible to select a period of time when we could stand this up?  Or should I just go ahead and work with our ops folks to stand up a long-living staging endpoint?

Reply:
@mquirion, how long can you keep the quick-server standing? If its a whole day, you can set it up on Monday and let me know. If its for a shorter duration, then we have to coordinate something definite since our timezones are different.

Reply:
@sumit116 you can use...

`{
  provider: 'staq',
  options: {
    url: 'nifi.staging.hdf.gke.staq-infra.com',
    connId: '1234'
   }
}`

Reply:
@sumit116 actually, you can use the following...

`options: {
                url: 'headerbidding.staging.staqdata.com',
                connId: '1234'
              }`

And that will get you hitting our staging server which we can leave open for a while.  Thank you.

Reply:
Thanks for providing these credentials; I gave them a try and found they were working.  I saw the events get sent to the staging endpoint.